books/RayTracingInOneWeekend.html

```diff
@@ -2011,19 +2011,18 @@
     [Listing [clamp]: <kbd>[interval.h]</kbd> The interval::clamp() utility function]
 
 And here's the updated `write_color()` function that takes the sum total of all light for the pixel
-and the number of samples involved:
+and a sampling scale value:
 
     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ C++ highlight
-    void write_color(std::ostream& out, const color& pixel_color, int samples_per_pixel) {
+    void write_color(std::ostream& out, const color& pixel_color, double spp_scale) {
         auto r = pixel_color.x();
         auto g = pixel_color.y();
         auto b = pixel_color.z();
 
-        // Divide the color by the number of samples.
-        auto scale = 1.0 / samples_per_pixel;
-        r *= scale;
-        g *= scale;
-        b *= scale;
+        // Divide the color by the sampling scale.
+        r *= spp_scale;
+        g *= spp_scale;
+        b *= spp_scale;
```

Comment:
Without a scale definition, the comment "Divide the color by the sampling scale." 
seems a _bit_ misleading.